pkg/api/convertertoapi.go

```diff
@@ -271,6 +271,10 @@ func convertVLabsAgentPoolProfile(vlabs *vlabs.AgentPoolProfile, api *AgentPoolP
 	api.VnetSubnetID = vlabs.VnetSubnetID
 	api.Subnet = vlabs.GetSubnet()
 	api.FQDN = vlabs.FQDN
+	api.CustomNodeLabels = map[string]string{}
```

Comment:
Does this work? You don't need to `make(map[string]string)` it? I can't remember in Go...